python/tvm/contrib/binutils.py

```diff
@@ -16,61 +16,10 @@
 # under the License.
 
 """Utilities for binary file manipulation"""
-import os
+import logging
 import subprocess
-import tvm._ffi
-from . import utils
 
-# TODO does this file still belong in `contrib`. is it too µTVM-specific?
-
-# TODO shouldn't need so many `ALIGN` directives
-RELOCATION_LD_SCRIPT_TEMPLATE = """
-/* linker symbol for use in UTVMInit */
```

Comment:
Is this change in the scope of the PR/RFC ? if not suggest we break it out to another PR.

Reply:
broken out